driver-core/src/main/java/com/datastax/driver/core/ProtocolVersion.java

```diff
@@ -30,10 +30,12 @@
     V1("1.2.0", 1),
     V2("2.0.0", 2),
     V3("2.1.0", 3),
+    V4("2.2.0", 4)
     ;
 
     /**
      * The most recent protocol version supported by the driver.
+     * TODO change to V4 when protocol is fully implemented.
```

Comment:
Since tests are now run on C* 2.2 by default, protocol V4 needs to be "hidden" until everything is implemented.